src/feature/control/control_cmd.c

```diff
@@ -299,12 +299,12 @@ handle_control_getconf(control_connection_t *conn,
       config_line_t *answer = option_get_assignment(options,q);
       if (!answer) {
         const char *name = option_get_canonical_name(q);
-        control_reply_add_1kv(answers, 250, KV_OMIT_VALS, name, "");
+        control_reply_add_one_kv(answers, 250, KV_OMIT_VALS, name, "");
       }
 
       while (answer) {
         config_line_t *next;
-        control_reply_add_1kv(answers, 250, KV_RAW, answer->key,
+        control_reply_add_one_kv(answers, 250, KV_RAW, answer->key,
                               answer->value);
```

Comment:
Nit: If you want to fix the spacing on this line before we merge, that would be good. Leaving it would also be ok.